django/db/models/query_utils.py

```diff
@@ -157,7 +157,7 @@ def deconstruct(self):
     def identity(self):
         path, args, kwargs = self.deconstruct()
         identity = [path, *kwargs.items()]
-        for child in args:
+        for child in sorted(args, key=str):
```

Comment:
Any thoughts about using `hash` instead?

Reply:
Sure, I could do that by moving the sort operation later, after all `make_hashable()` conversions. 👍